plot_krun_results.py

```diff
@@ -45,6 +45,14 @@
     'nbody':            'N-Body',
 }
 
+# Machine names that need to be reformatted.
```

Comment:
This is also true of the `BENCHMARKS` dict that you created. I agree a config file would be better, but I think we should deal with that in a separate PR and make all the `ALL_CAPS` globals configurable.